service_container/service_decoration.rst

```diff
@@ -48,22 +48,27 @@ the original service is lost:
         $container->register('app.mailer', DecoratingMailer::class);
 
 Most of the time, that's exactly what you want to do. But sometimes,
-you might want to decorate the old one instead. In this case, the
-old service should be kept around to be able to reference it in the
-new one. This configuration replaces ``app.mailer`` with a new one, but keeps
-a reference of the old one  as ``app.decorating_mailer.inner``:
+you might want to decorate the old service instead: keeping the old service so
```

Comment:
It's a bit weird to use two colons in this phrase. You could use a comma or a semicolon after `instead`